docs/tasks/2025-08-13-copy-as-prompt-feature.md

```
@@ -0,0 +1,304 @@
+  transcript?: string
+  lessons?: LessonData[]
+  contentType: 'post' | 'course' | 'lesson'
+  requiresPro?: boolean
+  className?: string
+  onCopy?: () => void
+}
+```
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)